lint/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@discourse/lint-configs",
```

Comment:
Nitpick: should we make the directory name match the package name? (i.e. `lint-configs/` instead of just `lint/` ?